.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "third_party/cpuinfo"]
+	path = third_party/cpuinfo
+	url = https://github.com/pytorch/cpuinfo
+[submodule "third_party/googletest"]
+	path = third_party/googletest
+	url = https://github.com/google/googletest
```

Comment:
You're probably going to want to update the Bazel workspace to use the deps from the submodule instead. Otherwise it's going to be really confusing that these don't match. Personally, I would add the submodules in a separate change, then switch Bazel over to it, then add the CMake stuff.

Also note that Copybara has a tendency to screw everything up when it comes to submodules, so you will need to double check that they actually got added properly

Reply:
thanks!